grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -247,10 +217,6 @@ class InvoiceItem implements Serializable {
                 isCanceled: orderItem?.canceled ?: orderAdjustment?.canceled,
                 quantityAvailableToInvoice: quantityAvailableToInvoice,
                 unitPriceAvailableToInvoice: orderAdjustment?.unitPriceAvailableToInvoice,
-                // Total amount and total prepayment amount are deprecated and amount field
-                // should be used instead (OBPIH-6398, OBPIH-6499)
-                totalAmount: totalAmount,
```

Comment:
I wonder if technically these two fields should stay for now, because this was expected as a part of `getPrepaymentItems`. I am not sure if we have a defined API endpoint deprecation flow. However, there is really high probability we were the only ones using it.

Reply:
so what's the decision here? should I revert those properties?